src/overlays/actors/ovl_En_Ending_Hero6/z_en_ending_hero6.c

```diff
@@ -24,20 +55,133 @@ const ActorInit En_Ending_Hero6_InitVars = {
     (ActorFunc)EnEndingHero6_Draw,
 };
 
-#endif
+static FlexSkeletonHeader* sSkeletons[] = { &D_0600B0CC, &D_06007908, &D_06007150, &D_0600D640, &D_0600A850,
+                                            &D_0600A850, &D_0600A850, &D_0600A850, &D_0600A850 };
+
+static AnimationHeader* sAnimations[] = { &D_06000BE0, &D_060011C0, &D_06000E50, &D_06002A84, &D_06002FA0,
+                                          &D_06002FA0, &D_06002FA0, &D_06002FA0, &D_06002FA0 };
+
+static s32 sLimbCounts[] = { 15, 20, 17, 17, 17, 17, 17, 17, 17 };
+
+void EnEndingHero6_Init(Actor* thisx, GlobalContext* globalCtx) {
+    EnEndingHero6* this = THIS;
+
+    this->actor.colChkInfo.mass = MASS_IMMOVABLE;
+    Actor_SetScale(&this->actor, 0.01f);
+    this->actor.targetMode = 6;
+    this->actor.gravity = -3.0f;
+    SkelAnime_InitSV(globalCtx, &this->skelAnime, sSkeletons[this->npcIndex], sAnimations[this->npcIndex],
+                     this->limbDrawTable, this->transitionDrawTable, sLimbCounts[this->npcIndex]);
+    ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 25.0f);
+    func_80C23DDC(this);
+}
+
+void EnEndingHero6_Destroy(Actor* thisx, GlobalContext* globalCtx) {
+}
+
+void func_80C23D60(EnEndingHero6* this, s32 npcIndex) {
+    this->animIndex = npcIndex;
+    this->frameCount = SkelAnime_GetFrameCount(&sAnimations[npcIndex]->common);
+    SkelAnime_ChangeAnim(&this->skelAnime, sAnimations[this->animIndex], 1.0f, 0.f, this->frameCount, 0, 0.0f);
```

Comment:
```suggestion
    SkelAnime_ChangeAnim(&this->skelAnime, sAnimations[this->animIndex], 1.0f, 0.0f, this->frameCount, 0, 0.0f);
```
Does it need have that extra 0 to be removed to match? There are rare cases where that's necessary, but if it still matches, this would be preferred

Reply:
Indeed the 0 must be removed from exactly one. I found that this function does not match if the two 0 float arguments are lexicographically equal.

Reply:
Amazing, that usually only happens in more complex functions. Then it's fine as is